ukvm/ukvm-configure

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ "$#" -lt 1 ]; then
+    echo "Usage: ukvm-configure UKVM_SRC [MODULES]"
+    echo "    UKVM_SRC is /path/to/ukvm"
+    echo "    MODULES can be any combination of: net blk gdb"
+    exit
+fi
+
+UKVM_SRC=`readlink -e $1`
+shift
+UKVM_MODULES=$@
+
+(
```

Comment:
@ricarkol @mato If I understand what you are saying @ricarkol, the `Makefile.ukvm` would be the same in all cases.  The only thing that would change is what `UKVM_MODULES` was set to before the `include Makefile.ukvm`.  Is that right?